src/Microsoft.AspNetCore.Rewrite/Internal/RedirectToWwwRule.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using Microsoft.AspNetCore.Rewrite.Logging;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.Rewrite.Internal
+{
+    public class RedirectToWwwRule : IRule
+    {
+        public int StatusCode { get; set; }
+        public bool AllowLocalhost { get; set; } = true;
+
+        public virtual void ApplyRule(RewriteContext context)
+        {
+            var req = context.HttpContext.Request;
+            if (AllowLocalhost && req.Host.Host.StartsWith("localhost", StringComparison.OrdinalIgnoreCase))
```

Comment:
I don't see why you need to do StartsWith here.

Reply:
As opposed to straight equals? I think you're right actually, I was using Value previously